dbsnap_verify/__main__.py

```diff
@@ -0,0 +1,14 @@
+import argparse
+
+from . import handler
+
+def main():
+    parser = argparse.ArgumentParser(description = 'verify RDS databases.')
+    parser.add_argument('config', help='The path to JSON config.')
```

Comment:
So this is a JSON config mostly because the Cloudwatch Event Rule Trigger uses JSON.  The JSON will automatically be turned into a `dict` named `event`when the lambda handler is invoked.  This config and cli tool allows us to bypass Lambda for testing but it isn't used at all when Lambda runs the code.